sdk/build.gradle.kts

```diff
@@ -99,7 +99,7 @@ tasks.withType<DokkaTask>().configureEach {
       if (buildFromSource.toBoolean()) {
         File("upstream-api-doc-list.txt").forEachLine {
           if (!it.startsWith("//")) {
-            sourceRoots.from(file("../../$it"))
+            sourceRoots.setFrom(file("$buildDir/../../../../src/android/java"))
```

Comment:
I guess it need to be applied to all the modules besides the sdk

Reply:
Well it's single place where we have such setup for Dokka actually